### ISSUE TITLE
Why are you using binary cross entropy

### ISSUE THREAD

**Issue by aribornstein** [bzamecnik/ml#46]
Hey this is really cool work. Is there are a reason you are using binary cross entropy loss over categorical cross entropy loss given that this seems to be a multiclass classification problem in this sample https://github.com/bzamecnik/ml/blob/master/chord-recognition/lstm_chord_classification_training.py

**Comment by bzamecnik** [comment]
Hi, it is multi-label classification, ie. N independent binary classifiers, each with sigmoid non-linearity and binary_crossentropy. Softmax would make it dependent and normalize it, so that there's a single class.
